<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContinueCompareClass</class>
 <widget class="QDialog" name="ContinueCompareClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>315</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>532</width>
    <height>315</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>532</width>
    <height>315</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Counter_ContinueCompare</string>
  </property>
  <widget class="QFrame" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>532</width>
     <height>321</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>532</width>
     <height>321</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>532</width>
     <height>321</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#background{background-image:url(:/ContinueCompare/Resources/Background.bmp)}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>49</y>
      <width>171</width>
      <height>215</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>171</width>
      <height>215</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>171</width>
      <height>215</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>152</width>
       <height>92</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>152</width>
       <height>92</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>152</width>
       <height>92</height>
      </size>
     </property>
     <property name="title">
      <string>Table 1</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>17</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>81</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>81</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>CompareData0:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtTab1Data0">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>16</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtTab1Data1">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>42</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>43</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>81</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>81</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>CompareData1:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtTab1Data2">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>66</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>67</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>81</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>81</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>CompareData2:</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>152</width>
       <height>92</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>152</width>
       <height>92</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>152</width>
       <height>92</height>
      </size>
     </property>
     <property name="title">
      <string>Table 2</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>17</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>81</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>81</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>CompareData0:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtTab2Data0">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>16</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtTab2Data1">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>42</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>43</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>81</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>81</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>CompareData1:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtTab2Data2">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>66</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>67</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>81</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>81</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>CompareData2:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>197</x>
      <y>49</y>
      <width>161</width>
      <height>215</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>215</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>161</width>
      <height>215</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>11</y>
       <width>141</width>
       <height>92</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>141</width>
       <height>92</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>141</width>
       <height>92</height>
      </size>
     </property>
     <property name="title">
      <string>Interval 1</string>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>17</y>
        <width>63</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>63</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>63</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>FirstValue:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtInt1FirstValue">
      <property name="geometry">
       <rect>
        <x>74</x>
        <y>16</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtInt1Increment">
      <property name="geometry">
       <rect>
        <x>74</x>
        <y>42</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>43</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Increment:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtInt1Count">
      <property name="geometry">
       <rect>
        <x>74</x>
        <y>66</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>67</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Count:</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>111</y>
       <width>141</width>
       <height>92</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>141</width>
       <height>92</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>141</width>
       <height>92</height>
      </size>
     </property>
     <property name="title">
      <string>Interval 2</string>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>17</y>
        <width>63</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>63</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>63</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>FirstValue:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtInt2FirstValue">
      <property name="geometry">
       <rect>
        <x>74</x>
        <y>16</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtInt2Increment">
      <property name="geometry">
       <rect>
        <x>74</x>
        <y>42</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>43</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Increment:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtInt2Count">
      <property name="geometry">
       <rect>
        <x>74</x>
        <y>66</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>67</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Count:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>131</width>
      <height>215</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>131</width>
      <height>215</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>131</width>
      <height>215</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="txtCounterValue">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>10</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>110</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>110</width>
       <height>25</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConfig">
     <property name="geometry">
      <rect>
       <x>34</x>
       <y>165</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Configure...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStart">
     <property name="geometry">
      <rect>
       <x>34</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStop">
     <property name="geometry">
      <rect>
       <x>34</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="radTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>42</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>101</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>101</width>
      <height>16</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Compare table</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radInterval">
    <property name="geometry">
     <rect>
      <x>207</x>
      <y>42</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>121</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>121</width>
      <height>16</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Compare interval</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>280</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>171</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>171</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string>Counter pattern match count:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPMCount">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>187</x>
      <y>277</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>25</height>
     </size>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>278</x>
      <y>281</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>141</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>141</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string>Compare table end count:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEndCount">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>278</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>25</height>
     </size>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="continuecompare.qrc"/>
 </resources>
 <connections/>
</ui>
